### PR TITLE
Added seeders for models

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Closes #XXXXX

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
Depends on https://github.com/freeCodeCamp/chapter/pull/299

File: server/factories/sponsor.factory.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const randomIndex = Math.floor(Math.random() * enumValues.length);
+  const randomEnumValue = anEnum[randomIndex];
+  return randomEnumValue;
+}
+
+define(Sponsor, (faker: typeof Faker) => {
+  const name = faker.company.companyName();
+  const website = faker.internet.url();
+  const logoPath = faker.system.commonFileName('png');
+  enum SponsorType {
```
Comment (Zeko369) [inline server/factories/sponsor.factory.ts]: Export this from types/something

File: server/factories/sponsor.factory.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import Faker from 'faker';
+import { define } from 'typeorm-seeding';
+
+import { Sponsor } from '../models/Sponsor';
+
+function randomEnum<T>(anEnum: T): T[keyof T] {
```
Comment (Zeko369) [inline server/factories/sponsor.factory.ts]: Use arrow functions, and also maybe extract this into a helper

**Comment by Zeko369** [comment]
image
I'm getting this? Is it working on your machine?

**Comment by vaibhavsingh97** [comment]
yes, it's working on my machine, can you please drop db and retry once again?

**Pull request merged**